test/unit/test_unit_tests.py

```diff
@@ -397,10 +396,11 @@ def test_getting_dependencies_gradle(self) -> None:
             failed_assertion_message = 'failed for app = ' + app_name
             self.assertTrue(generated_classpath != '', failed_assertion_message)
             self.assertTrue(os.path.isfile(generated_classpath), failed_assertion_message)
-            self.__assert_classpath(standard_classpath,
-                                    generated_classpath,
-                                    os.path.join(dir_util.get_output_dir(app_name, config['general'].get('module_name', '')), dependencies_dir),
-                                    failed_assertion_message)
+            self.__assert_classpath(standard_classpath=standard_classpath,
+                                    generated_classpath=generated_classpath,
+                                    std_classpath_prefix='',
+                                    message=failed_assertion_message,
+                                    ordered_classpath=True)
```

Comment:
is ordered_classpath a temporary variable until all build types preserve order of jars in classpath? need to remember to remove it once support for ant is added

Reply:
yes, when we add ant support for jar orders (hope it will be disposable) we will remove the "ordered_classpath" option.
we will also will remove code from the option_util.py.
will add a comment